prow/cluster/labelsync_cronjob.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         spec:
           containers:
             - name: label-sync
-              image: eu.gcr.io/jetstack-build-infra/prow-labelsync:latest
+              image: gcr.io/k8s-prow/label_sync:v20230405-89944e7c3e
```

Comment:
There is probably a huge diff here as the last time labelsync was pushed to our registry was in 2019 https://console.cloud.google.com/gcr/images/jetstack-build-infra/eu/prow-labelsync?authuser=1&project=jetstack-build-infra
Please take a look at the release notes and add a note whether there is any diff in how it works that would affect us.

Reply:
@irbekrm where do I find these release notes? Do I manually check the git diff?

Reply:
I normally just skim through Prow announcements, I see that this is now moved to https://docs.prow.k8s.io/docs/announcements/